src/iologindata.cpp

```diff
@@ -470,6 +472,54 @@ bool IOLoginData::loadPlayer(Player* player, DBResult_ptr result)
 		}
 	}
 
+	//load reward chest items
+	itemMap.clear();
+
+	query.str(std::string());
+	query << "SELECT `pid`, `sid`, `itemtype`, `count`, `attributes` FROM `player_rewards` WHERE `player_id` = " << player->getGUID() << " ORDER BY `sid` DESC";
+	if ((result = db->storeQuery(query.str()))) {
```

Comment:
@WibbenZ compilers issue a warning when such attribution is not enclosed in parenthesis as it may solve some ambiguity problems.